tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -63,12 +63,12 @@ public partial class SpanContext : ISpanContext, IReadOnlyDictionary<string, str
         /// Initializes a new instance of the <see cref="SpanContext"/> class
         /// from a propagated context. <see cref="Parent"/> will be null
         /// since this is a root context locally.
-        /// Internal for testing
         /// </summary>
         /// <param name="traceId">The propagated trace id.</param>
         /// <param name="spanId">The propagated span id.</param>
         /// <param name="samplingPriority">The propagated sampling priority.</param>
         /// <param name="serviceName">The service name to propagate to child spans.</param>
+        [InternalForTesting]
```

Comment:
This is `public`, not `internal`.

Reply:
Fixed by changing the name of the attribute